dist/app.js

```diff
@@ -4,14 +4,12 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 const express_1 = __importDefault(require("express"));
-const body_parser_1 = __importDefault(require("body-parser"));
 require("dotenv/config");
+const phones_1 = require("./Controllers/phones");
 const app = (0, express_1.default)();
 const port = process.env.PORT;
-app.use(body_parser_1.default.json());
-app.get('/', (req, res) => {
-    res.send('Hello, world!');
-});
+app.get('/products', phones_1.getAllPhones);
```

Comment:
app.get('/phones', phones_1.getAllPhones);
app.get('/phones/:phoneId', phones_1.getPhoneById);
rename to phones